### PR TITLE
Changed style of enter grade link

### PR DESCRIPTION
**Pull request by amir-qayyum-arbisoft**
Changed enter grade link style to make it look like button.
#15

### CODE REVIEW

**Comment by carsongee** [comment]
For UI changes, it would be handy to have a screen shot with the PR in the future.

**Comment by carsongee** [comment]
Since the goal is to make this look like the other button, is there any reason you didn't just switch this to an input button and use the same class as the annotated button?  Also, as is the size is slightly off between the two:
image

**Comment by amir-qayyum-arbisoft** [comment]
Hi

1) Annotated button use class="fileupload" and with this class we trigger file upload function
2) Annotated button code is 
      <input class="fileupload" type="file" name="annotated"/>
       <button>Upload annotated file</button>

Input is placed inside button, i dont now why?

3) Enter grade use href="#{{ id }}-enter-grade" to open dialog

**Comment by amir-qayyum-arbisoft** [comment]
fixed the size of button

**Comment by carsongee** [comment]
Thanks for the explanation.  I think we should then create a common class for both i.e. `.sga-block staff-button` and have both those change to use the new class.  Please also switch the Enter Grade a tag to an input button so they have consistent HTML as well

**Comment by amir-qayyum-arbisoft** [comment]
@carsongee  I am working on this PR

**Comment by pdpinch** [comment]
@amir-qayyum-arbisoft I just noticed you pushed a bunch of commits with the same commit message. That makes it really hard to figure out what you're doing. I know that sometimes you can end up with more commits than you intended -- in those situations you can squash them ([edX has docs on this](https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request#squash-your-changes-optional); I frequently use [this guide](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) myself)

**Comment by amir-qayyum-arbisoft** [comment]
@pdpinch  i will gona sqash these commits, just testing different scanarios

File: edx_sga/templates/staff_graded_assignment/show.html [hunk 1]
```diff
@@ -87,13 +87,13 @@
...
           <td>
             <% if (assignment.may_grade) { %>
-              <a class="enter-grade-button" href="#{{ id }}-enter-grade">
-                <% if (assignment.needs_approval) { %>
-                  {% trans "Approve grade" %}
-                <% } else { %>
-                  {% trans "Enter grade" %}
-                <% } %>
-              </a>
+                 <a class="enter-grade-button button" href="#{{ id }}-enter-grade">
```
Comment (carsongee) [inline edx_sga/templates/staff_graded_assignment/show.html]: nit: two space index

**Comment by carsongee** [comment]
There is still a slight size difference:
image
but once that is fixed and merge conflicts are resolved I'm :+1:

**Comment by amir-qayyum-arbisoft** [comment]
@carsongee  done with changes and rebase

File: edx_sga/static/css/edx_sga.css [hunk 2]
```diff
@@ -119,7 +121,32 @@
...
+    white-space: nowrap;
+    top: 0px;
+    left: 0px;
+    z-index: 2;
+    padding: 2px;
+    width: 200px;
+    color: #333346;
+    font-weight: bold;
+    font-size: 0.8125em;
+    margin-top: 1px;
```
Comment (carsongee) [inline edx_sga/static/css/edx_sga.css]: This makes the button one pixel lower than the other button.  I think that if you remove this line this is definitely good to go.  Sorry to be so particular:
image

**Comment by amir-qayyum-arbisoft** [comment]
@carsongee done with 1px fix 
enter_grade_link_fix

**Comment by carsongee** [comment]
Awesome, thanks so much @amir-qayyum-arbisoft :sparkler:

**Pull request merged**